lib/node_modules/@stdlib/blas/base/gnrm2/test/test.main.js

```diff
@@ -87,32 +105,102 @@ tape( 'the function supports a `stride` parameter', function test( t ) {
 		2.0
 	];
 
-	N = 4;
-	z = gnrm2( N, x, 2 );
+	z = gnrm2( 4, x, 2 );
 
 	t.strictEqual( z, 5.0, 'returns expected value' );
 	t.end();
 });
 
-tape( 'if provided a `stride` parameter less than or equal to `0`, the function returns `0`', function test( t ) {
+tape( 'the function supports a `stride` parameter (accessors)', function test( t ) {
+	var x;
+	var z;
+
+	x = [
+		1.0,  // 0
+		2.0,
+		2.0,  // 1
+		-7.0,
+		-2.0, // 2
+		3.0,
+		4.0,  // 3
+		2.0
+	];
+
+	z = gnrm2( 4, toAccessorArray( x ), 2 );
+
+	t.strictEqual( z, 5.0, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function supports a negative `stride` parameter', function test( t ) {
+	var x;
+	var z;
+
+	x = [
+		1.0,  // 3
+		2.0,
+		2.0,  // 2
+		-7.0,
+		-2.0, // 1
+		3.0,
+		4.0,  // 0
+		2.0
+	];
+
+	z = gnrm2( 4, x, -2 );
+
+	t.strictEqual( z, 5.0, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function supports providing a negative `stride` parameter (accessors)', function test( t ) {
+	var x;
+	var z;
+
+	x = [
+		1.0,  // 3
+		2.0,
+		2.0,  // 2
+		-7.0,
+		-2.0, // 1
+		3.0,
+		4.0,  // 0
+		2.0
+	];
+
+	z = gnrm2( 4, x, -2 );
+
+	t.strictEqual( z, 5.0, 'returns expected value' );
+	t.end();
+});
+
+tape( 'if provided a `stride` parameter equal to `0`, the function returns the L2-norm of the first element of the vector multiplied by `N` square root', function test( t ) {
```

Comment:
```suggestion
tape( 'if provided a `stride` parameter equal to `0`, the function returns the L2-norm of a vector', function test( t ) {
```

Your definition is not correct. Better to avoid fitting a square peg into a round hole on this one.